js/HelloWorldSceneAR.js

```diff
@@ -1,35 +1,59 @@
 import React, { Component } from "react";
+import {StyleSheet} from 'react-native';
 import {
   ViroARScene,
   Viro3DObject,
   ViroARImageMarker,
   ViroARTrackingTargets,
-  ViroAmbientLight
+  ViroAmbientLight,
+  ViroText,
 } from "react-viro";
+import AssetLoader from './AssetLoader';
 
-ViroARTrackingTargets.createTargets({
-  qrcode: {
-    source: require("./res/qrcode.png"),
-    orientation: "Up",
-    physicalWidth: 0.1
+export default class HelloWorldSceneAR extends Component {
+
+  constructor() {
+    super();
+    this.assetLoader = new AssetLoader('https://skvngr-server.herokuapp.com');
```

Comment:
You don't need to put it into the constructor. You can just use class properties here.

```suggestion
  assetLoader = new AssetLoader('https://skvngr-server.herokuapp.com');
```